backend/compose.yaml

```diff
@@ -4,4 +4,24 @@ services:
       context: .
       dockerfile: ./Containerfile
     ports:
-      - 8080:8080 # API
+      - 8080:8080
+    depends_on:
+      - db
+    environment:
+      - DB_HOST=db
+      - DB_PORT=5432
+      - DB_USER=testuser
+      - DB_PASSWORD=testpassword
+      - DB_NAME=testdb
+
+  db:
+    image: postgres:15
+    environment:
+      POSTGRES_USER: testuser
+      POSTGRES_PASSWORD: testpassword
+      POSTGRES_DB: testdb
+    ports:
+      - 5432:5432
+    volumes:
+      - ./db_data:/var/lib/postgresql/data
```

Comment:
Just wondering why are we using a bind mount here.

Reply:
A binding mount for `./db_data:/var/lib/postgresql/data` allows for local persistent data storage even if the container is stopped or removed.

Reply:
If that is the case, would a named volume serve our purpose?